// [[Reason: Sourcemap is more detailed]] ////

//// [tests/cases/compiler/declarationEmitInferredDefaultExportType2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,6 +1,6 @@
 
 //// [declarationEmitInferredDefaultExportType2.d.ts.map]
-{"version":3,"file":"declarationEmitInferredDefaultExportType2.d.ts","sourceRoot":"","sources":["declarationEmitInferredDefaultExportType2.ts"],"names":[],"mappings":";;;;;AACA,kBAIC"}
+{"version":3,"file":"declarationEmitInferredDefaultExportType2.d.ts","sourceRoot":"","sources":["declarationEmitInferredDefaultExportType2.ts"],"names":[],"mappings":";IAEE,GAAG;IACH,GAAG;IACH,GAAG;;AAHL,kBAIC"}
 
-//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBfZGVmYXVsdDogew0KICAgIGZvbzogcmVhZG9ubHkgW107DQogICAgYmFyOiBhbnk7DQogICAgYmF6OiBhbnk7DQp9Ow0KZXhwb3J0ID0gX2RlZmF1bHQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1kZWNsYXJhdGlvbkVtaXRJbmZlcnJlZERlZmF1bHRFeHBvcnRUeXBlMi5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0SW5mZXJyZWREZWZhdWx0RXhwb3J0VHlwZTIuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xhcmF0aW9uRW1pdEluZmVycmVkRGVmYXVsdEV4cG9ydFR5cGUyLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7O0FBQ0Esa0JBSUMifQ==,Ly8gdGVzdC50cwpleHBvcnQgPSB7CiAgZm9vOiBbXSBhcyBjb25zdCwKICBiYXI6IHVuZGVmaW5lZCwKICBiYXo6IG51bGwKfQ==
+//// https://sokra.github.io/source-map-visualization#base64,ZGVjbGFyZSBjb25zdCBfZGVmYXVsdDogew0KICAgIGZvbzogcmVhZG9ubHkgW107DQogICAgYmFyOiBhbnk7DQogICAgYmF6OiBhbnk7DQp9Ow0KZXhwb3J0ID0gX2RlZmF1bHQ7DQovLyMgc291cmNlTWFwcGluZ1VSTD1kZWNsYXJhdGlvbkVtaXRJbmZlcnJlZERlZmF1bHRFeHBvcnRUeXBlMi5kLnRzLm1hcA==,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiZGVjbGFyYXRpb25FbWl0SW5mZXJyZWREZWZhdWx0RXhwb3J0VHlwZTIuZC50cyIsInNvdXJjZVJvb3QiOiIiLCJzb3VyY2VzIjpbImRlY2xhcmF0aW9uRW1pdEluZmVycmVkRGVmYXVsdEV4cG9ydFR5cGUyLnRzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7SUFFRSxHQUFHO0lBQ0gsR0FBRztJQUNILEdBQUc7O0FBSEwsa0JBSUMifQ==,Ly8gdGVzdC50cwpleHBvcnQgPSB7CiAgZm9vOiBbXSBhcyBjb25zdCwKICBiYXI6IHVuZGVmaW5lZCwKICBiYXo6IG51bGwKfQ==
 
